go/v4/exchange_crypto_256k1_windows.go

```diff
@@ -1,5 +1,3 @@
-//go:build windows
-// +build windows
```

Comment:
do we still need this file?

Reply:
@ttodua I'm leaving it for now just in case we need to revert this fix quickly